CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contribution Guidelines
+
+Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms. Please also review our [Contributor License Agreement ("CLA")](INDIVIDUAL_CONTRIBUTOR_LICENSE.md) prior to submitting changes to the project.  You will need to attest to this agreement following the instructions in the [Paperwork for Pull Requests](#paperwork-for-pull-requests) section below.
+
+---
+
+# How to Contribute
+
+Now that we have the disclaimer out of the way, let's get into how you can be a part of our project. There are many different ways to contribute.
+
+## Issues
+
+We track our work using Issues in GitHub. Feel free to open up your own issue to point out areas for improvement or to suggest your own new experiment. If you are comfortable with signing the waiver linked above and contributing code or documentation, grab your own issue and start working.
+
+## Coding Standards
+
+We have some general guidelines towards contributing to this project.
+
+### Languages
+
+*Lua*
+
+## Pull Requests
+
+If you've gotten as far as reading this section, then thank you for your suggestions.
+
+## Paperwork for Pull Requests
+
+* Please read this guide and make sure you agree with our [Contributor License Agreement ("CLA")](INDIVIDUAL_CONTRIBUTOR_LICENSE.md).
+* Make sure git knows your name and email address:
+   ```
+   $ git config user.name "J. Random User"
+   $ git config user.email "j.random.user@example.com"
+   ```
+>The name and email address must be valid as we cannot accept anonymous contributions.
+* Write good commit messages.
+> Concise commit messages that describe your changes help us better understand your contributions.
+* The first time you open a pull request in this repository, you will see a comment on your PR with a link that will allow you to sign our Contributor License Agreement (CLA) if necessary.
```

Comment:
Is there a github workflow for this?

Reply:
There will be once the CLA assistant is turned on at the organization level. Part of this commit has the INDIVIDUAL_CONTRIBUTOR_LICENSE.md file which the CLA assistant will use as a digital contract. That should only show up the first time someone is contributing to the repo under a username.